moped-editor/src/components/GridTable/Filters.js

```diff
@@ -108,38 +108,35 @@ const useStyles = makeStyles((theme) => ({
  * @constructor
  */
 const Filters = ({
-  filters,
   setFilters,
   handleAdvancedSearchClose,
   filtersConfig,
   resetSimpleSearch,
   isOr,
   setIsOr,
+  searchParams,
+  setSearchParams,
```

Comment:
im not sure exactly why but sharing these with the Search parent component and passing them down from there fixed some issues with the URL search params not clearing on Reset button click